CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## Next Release
 ### Features
-- [In Progress: 29 Jun 2021] Add support for finding a patient online from ID scan
+- [In Progress: 29 Jun 2021]
+  - Add support for finding a patient online from ID scan
+  - Delete patients when the retention time has passed
```

Comment:
We can move this to internal, `Add support for finding a patient online from ID scan` is the feature. So any other change that is not visible to the user that is not covered by the above mentioned changelog doesn't have to go here.